org.gnome.EasyTAG.appdata.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be nice to submit these changes upstream.

Reply:
Why there's a new appdata long with a patch to apply on top of the current shipped one?

Reply:
Apparently with the 3.38 SDK and the new intltool module there's a check that runs on the appdata.xml generated from the tarball, which stops the build process, so the patch is to get around that, while the final appdata.xml is still the external one. The stable release is quite old (released in December 2016), and I'm using the tarball here, so if there's a better way please let me know.

Reply:
Turns out that those rename options actually do more than that, they also rename the ids of things in the files, so the only modification needed is that patch file to add the releases and content_rating to the appdata.xml. I have submitted a PR to upstream [here](https://gitlab.gnome.org/GNOME/easytag/-/merge_requests/9).